web/static/js/components/idea_generation_lower_third.jsx

```diff
@@ -12,10 +12,12 @@ const IdeaGenerationLowerThird = props => {
 
   const isFacilitator = currentUser.is_facilitator
   const stageConfig = stageConfigs[stage]
-  const showActionItem = stage !== "idea-generation"
+  const showActionItem = !["idea-generation", "voting"].includes(stage)
 
-  function wereActionItemsSubmitted() {
-    return showActionItem && !ideas.some(idea => idea.category === "action-item")
+  function progressionDisabled() {
```

Comment:
This was a bug fix implemented per Sean's suggestion. He was under the impression that the progression button should be disabled until an idea has been submitted. Should we create a bug fix PR for this?